int/hwsc-file-transaction-svc/hwsc-file-transaction-svc.js

```diff
@@ -31,6 +32,32 @@ function getStatus(callback) {
   });
 }
 
+function uploadFile(callback) {
+  if (typeof callback !== 'function') {
+    console.error('callback not a function');
+    return;
+  }
+  // 'utf8'
+  fs.readFile('res/cat.jpg', 'binary', (errFile, contents) => {
```

Comment:
Consider using ``fs.createReadStream``. According to my research, the usage of ``binary`` arg here is wrong. Please review the issue ticket.